src/components/container/DeviceDataMonthCharts/DeviceDataMonthCharts.tsx

```diff
@@ -114,6 +114,17 @@ export default function (props: DeviceDataMonthChartsProps) {
 					month={month}
 					year={year}
 					syncId="DeviceDataCharts" />
+				<DeviceDataMonthChart onDataDetected={() => onDataDetected()}
```

Comment:
Can probably remove this chart from this standard viz, we aren't really using the DeviceDataMonthChart in view builder etc.